python/paddle/fluid/tests/unittests/test_lrn_op.py

```diff
@@ -87,5 +87,15 @@ def test_check_grad_normal(self):
         self.check_grad(['X'], 'Out', max_relative_error=0.01)
 
 
+class TestLRNMKLDNNOp(TestLRNOp):
```

Comment:
There are two algorithms ACROSS_CHANNELS and WITHIN_CHANNEL, but TestLRNMKLDNNOp here only test the default one.

Reply:
Thanks for this remark, @luotao1. PaddlePaddle seems to implement LRN that works across channels. For MKLDNN, WITHIN_CHANNEL option does not work in backward pass.

I decided to remove "algorithm" attribute and fix LRN algorithm to ACROSS_CHANNELS.